.dockerignore

```diff
@@ -0,0 +1,7 @@
+ve/
+
+build
+dist
+*.egg-info
+*.egg/
+*.pyc
```

Comment:
I think the important things are:
```
.git
.gitignore

.dockerignore
Dockerfile

sideloader
.deploy.yaml

build
dist
*.egg-info
*.egg
**/*.pyc
```

A couple of things...
* It's important to not include `.git` or else any `git` command will invalidate the cache.
* Unlike .gitignores, .dockerignores (annoyingly) aren't recursive by default. So `**/*.pyc` is needed to remove all `.pyc` files.
* We add `Dockerfile` to the .dockerignore so that minor changes to the Dockerfile don't cause everything to be recopied. It's also not necessary to have the Dockerfile inside the image it has built.

It's annoyingly tricky to get `COPY . /app` to work just right. The other option is to copy in each file/directory you need. But this is probably the best still :-/